transcoder/basisu.h

```diff
@@ -264,7 +264,7 @@ namespace basisu
 	template<uint32_t NumBytes>
 	struct packed_uint
 	{
-		uint8_t m_bytes[NumBytes];
+		uint8_t m_bytes[4];
```

Comment:
Although it's not clear if this is a "public" member, I figure that since the code static assert in the constructor on size <= 4, that this should be ok.

ONly problem I see is if this member is public, and used with `sizeof()`

Reply:
I can't take this PR - the structure is variable size. So if NumBytes==2, it should be a 2 byte struct. ("#pragma pack(1)" is commonly used with this struct.)